src/components/Navigation/Ui/SearchBar.js

```diff
@@ -6,7 +6,7 @@ export default class SearchBar extends Component {
   render() {
     const { filterOptions, onChange, onSearch } = this.props
     return (
-      <Row className="show-grid spacer">
+      <Row className="show-grid spacer-search">
```

Comment:
Changing a className here could affect the e2e tests, please check if everything is fine.